cmd/syncthing/gui.go

```diff
@@ -1024,6 +1027,87 @@ func (s *apiService) getSystemLogTxt(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+type fileEntry struct {
+	name string
+	data []byte
+}
+
+func (s *apiService) getSupportBundle(w http.ResponseWriter, r *http.Request) {
+	var arrayFile []fileEntry
```

Comment:
It's not an array technically, it's a slice, so either just call it files, or perhaps use a map, given it's a name to bytes type of struct.